main.go

```diff
@@ -43,33 +47,40 @@ func main() {
 
 }
 
-func generateDocs() {
-	// schema := provider.Provider().Schema
+func generateDocs() error {
+
+	docsPath := "docs/resources"
+
 	resources := provider.Provider().ResourcesMap
 
-	names := make([]string, 0)
-	for k := range resources {
-		names = append(names, k)
-	}
-	sort.Strings(names)
-	for _, name := range names {
-		resource := resources[name]
-		fmt.Printf("\n### %s\n\n", name)
+	for name, resource := range resources {
+		shortName := strings.TrimPrefix(name, "snowflake_")
+
+		f, err := os.Create(path.Join(docsPath, fmt.Sprintf("%s.md", shortName)))
+		if err != nil {
+			return err
+		}
+		defer f.Close()
+
+		f.WriteString(fmt.Sprintf("\n# %s\n\n", name))
```

Comment:
**[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Error return value of `f.WriteString` is not checked (errcheck)